README.md

```diff
@@ -41,26 +54,9 @@ Both of these items must be provided from a Kubernetes secret. The secret must b
 }
 ```
 
-If access to private registries is not needed, the `dockercfg.json` file contents should look like this:
-
-```json
-{}
-```
-
-2. Locate your Snyk Integration ID from the Snyk Integrations page (navigate to https://app.snyk.io/org/YOUR-ORGANIZATION-NAME/manage/integrations/kubernetes) and copy it.
-The Snyk Integration ID is a UUID and looks similar to the following:
-```
-abcd1234-abcd-1234-abcd-1234abcd1234
-```
-The Snyk Integration ID is used in the `--from-literal=integrationId=` parameter in the next step.
-
-3. Finally, create the secret in Kubernetes by running the following command:
-```shell
-kubectl create secret generic snyk-monitor -n snyk-monitor --from-file=./dockercfg.json --from-literal=integrationId=abcd1234-abcd-1234-abcd-1234abcd1234
-```
-
 Note that the secret _must_ be namespaced, and the namespace (which we configured earlier) is called _snyk-monitor_.
```

Comment:
This line feels like it is now too further down at the bottom, could we move it somewhere around step 2, maybe before "Continue to YAML files installation instructions below."?
Or alternatively just delete it, I don't see a similar instruction in our public docs, which should be the most accurate

Reply:
I'm deleting! :)